Corona-Warn-App/src/main/java/de/rki/coronawarnapp/storage/TracingRepository.kt

```diff
@@ -70,6 +78,10 @@ class TracingRepository @Inject constructor(
         }
     }
 
+    private fun List<TaskInfo>.isRiskLevelTaskRunning() = find {
```

Comment:
I think there is a method called `List<T>.any { }` which returns true if an of the elements match the condition.